definitions/json/AuthorisationCaseEvent/AuthorisationCaseEvent-ET1Repped.json

```diff
@@ -16,6 +16,7 @@
   {"CaseTypeId": "ET_EnglandWales", "CaseEventID": "et3ResponseDetails", "UserRole": "[CLAIMANTSOLICITOR]", "CRUD": "D"},
   {"CaseTypeId": "ET_EnglandWales", "CaseEventID": "submitEt3", "UserRole": "[CLAIMANTSOLICITOR]", "CRUD": "D"},
   {"CaseTypeId": "ET_EnglandWales", "CaseEventID": "respondentTSE", "UserRole": "[CLAIMANTSOLICITOR]", "CRUD": "D"},
+  {"CaseTypeId": "ET_EnglandWales", "CaseEventID": "claimantTSE", "UserRole": "[CLAIMANTSOLICITOR]", "CRUD": "D"},
```

Comment:
Should this not be CRUD?
A claimant solicitor should have access to this event?

Reply:
resolved in https://github.com/hmcts/et-ccd-definitions-englandwales/pull/946/commits/4f772d528784427c0849821e05d607efda577c1a